apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -0,0 +1,247 @@
+        })}
+        <script>
+          $(function () {
+            $('input[name=cr-role]').change(function () {
+              var role = this.value;
+              $('.question-form button').prop('disabled', role != 'instructor');
+              $('.role-comment').hide();
+              $('.role-comment-' + role).show();
+            });
+          });
+        </script>
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../partials/navbar')%>", {
+          ...resLocals,
+          navPage: 'request_course',
+        })}
+        <main id="content" class="container">
+          ${course_requests.length > 0
+            ? html`
+                <!-- Previous Course Requests -->
+                <div class="card mb-4">
+                  <div class="card-header bg-primary text-white d-flex align-items-center">
+                    Course Requests
+                  </div>
+                  <div class="table-responsive">
+                    <table class="table table-sm table-hover table-striped">
+                      <thead>
+                        <tr>
+                          <th>Short Name</th>
+                          <th>Title</th>
+                          <th>Status</th>
+                          <th>Details</th>
+                        </tr>
+                      </thead>
+                      <tbody>
+                        ${course_requests.map((req) => {
+                          let details = '';
+                          switch (req.course_request.approved_status) {
+                            case 'approved':
+                              if (req.approved_by_user) {
+                                details = `Approved by ${req.approved_by_user?.name}`;
+                              } else {
+                                details = 'Automatically approved';
+                              }
+                              break;
+                            case 'denied':
+                              details = `Denied by ${req.approved_by_user?.name}`;
+                              break;
+                          }
+
+                          return html`
+                            <tr>
+                              <td>${req.course_request.short_name}</td>
+                              <td>${req.course_request.title}</td>
+                              <td>
+                                ${renderEjs(__filename, "<%- include('approvalStatusIcon')%>", {
+                                  status: req.course_request.approved_status,
+                                })}
+                              </td>
+                              <td>${details}</td>
+                            </tr>
+                          `;
+                        })}
+                      </tbody>
+                    </table>
+                  </div>
+                </div>
+              `
+            : ''}
+
+          <!-- New Request -->
+          <div class="card mb-4">
```

Comment:
Also stick this card in a helper function and include it here. This will remove the need for the `<!-- New Request -->` comment.

Reply:
Changed in 9664c3b